CHANGES.md

```diff
@@ -70,6 +70,8 @@ To be released.
  -  The difficulty was changed from representing the number of leading zeros of
     target number to representing a divisor to obtain the target number.
     [[#213]]
+ -  Removed unnecessary reader lock on `BlockChain<T>.StageTrasactions()`. [[#217]]
```

Comment:
Was the removed thing a reader lock?  Wasn't it a writer lock?

Reply:
You're right. it was a writer lock.